### PR TITLE
Add task page

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/517

### CODE REVIEW

File: src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        [Parameter]
+        public Guid TaskId { get; set; }
+
+        private TasksResponse _task;
+
+        [Inject]
+        private IProjectService _ProjectService { get; set; }
+
+        // protected override async Task OnInitialized()
```
Comment (BelehovEgor) [inline src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor.cs]: Why did you comment it?

File: src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                                            </div>
+
+                                        @* @foreach (var item in collection)
+                                        {
+                                            <div class="subtasks-content_table-body">
+                                                <p>Create db models</p>
+                                                <p>20-07-2021</p>
+                                            </div>
+                                        } *@
+
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor:109]: Why did you not remove it?

**Review by MariyaSin — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

File: src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor.cs (hunk 1 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor.cs]: Reworked

File: src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor (hunk 2 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/TaskWindow/TaskModalWindow.razor:109]: Reworked

**Pull request merged**